### PR TITLE
[Snackbar] Add variable to change font color

### LINKED ISSUE #1278: [Snackbar] Inconsistent use of variable
https://github.com/buefy/buefy/blob/9170ec901a0c7643e080e9c5d171e0a088c6468d/src/scss/components/_notices.scss#L43

It should be like `$snackbar-background-color: $dark !default;`.
Maybe: `$snackbar-color: $dark-invert !default;`

This can be useful when we create a dark theme and want to display snackbar with light background.

### PR DESCRIPTION
**Pull request by Heziode**
And use `$snackbar-background-color` for the background of action button

Close #1278

### CODE REVIEW

**Comment by MattStellify** [comment]
👍

File: src/scss/components/_notices.scss [hunk 1]
```diff
@@ -52,6 +53,10 @@ $snackbar-background-color: $dark !default;
             .button {
                 font-weight: $weight-semibold;
                 text-transform: uppercase;
+                background: $snackbar-background-color;
+            }
+            .button:hover {
+                background: darken($snackbar-background-color, 10);
```
Comment (jtommy) [inline src/scss/components/_notices.scss]: It's better 5% instead of 10

Comment (Heziode) [reply src/scss/components/_notices.scss]: It is done.

Comment (jtommy) [reply src/scss/components/_notices.scss]: Please add % too :)

Comment (Heziode) [reply src/scss/components/_notices.scss]: Done ;)

**Comment by jtommy** [comment]
Thanks

**Pull request merged**